### PR TITLE
bt-591: Connect employer onboarding profile to api

### LINKED ISSUE #591: Connect Employer onboarding and profile to API

### LINKED ISSUE #640: Change a placeholder for the profile photo and company logo on the Employer’s onboarding page
Use these as a placeholder. Color should be suited to the design
https://fontawesome.com/icons/image?f=classic&s=solid
https://fontawesome.com/v5/icons/user-circle?f=classic&s=solid&pc=%23335a9e

Image
Image

### LINKED ISSUE #643: Add placeholder for Talent profile photo on step 04/talent profile
Use this as a placeholder. Color should be suited to the design
https://fontawesome.com/v5/icons/user-circle?f=classic&s=solid&pc=%23335a9e

Image

### LINKED ISSUE #644: Add a placeholder for the profile photo and company logo on the Employer’s onboarding page
Use these as a placeholder. Color should be suited to the design
https://fontawesome.com/icons/image?f=classic&s=solid
https://fontawesome.com/v5/icons/user-circle?f=classic&s=solid&pc=%23335a9e

Image

Image

### PR DESCRIPTION
**Pull request by anogire**
Screenshot 2023-09-25 at 03 49 56Screenshot 2023-09-25 at 03 50 03
Screenshot 2023-09-25 at 03 50 14Screenshot 2023-09-25 at 03 56 54

### CODE REVIEW

File: mobile/src/bundles/common/common-api.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ class TalentApi extends HttpApiBase {
 
     public async getUserDetailsByUserId(
         payload: Partial<UserDetailsGeneralRequestDto>,
-    ): Promise<UserDetailsGeneralRequestDto | null> {
+    ): Promise<UserDetailsResponseDto | null> {
         const { userId = '' } = payload;
         const response = await this.load(
             this.getFullEndpoint('/', userId, {}),
```
Comment (mskmee) [inline mobile/src/bundles/common/common-api.ts:58]: Are you sure about this slash? We have removed all shashes at the end of endpoints

**Comment by mskmee** [comment]
Don't know if it's a good idea to create a common data API, and use it to employer and talent.
We create unnecessary connexion between those bundles adn violated open/closed principles in talent store.
As for me, we should use employer slice as a separate slice. Better for logic and scaling application.
@katerynakostikova

**Comment by anogire** [comment]
I've already moved it to a common folder and made it one for talent and employer.

File: mobile/src/bundles/common/common-api.ts (hunk 1 above)
Comment (anogire) [reply mobile/src/bundles/common/common-api.ts:58]: This is a '/' before userId

**Review by mskmee — APPROVED** [review]

File: mobile/src/bundles/chat/screens/chat-list/chat-list.tsx [hunk 2]
```diff
@@ -78,10 +86,32 @@ const ChatList: React.FC = () => {
...
+                    globalStyles.p25,
+                    globalStyles.pr15,
+                    globalStyles.flexDirectionRow,
+                    globalStyles.justifyContentSpaceBetween,
+                    globalStyles.alignItemsCenter,
+                    styles.header,
+                ]}
+            >
                 <Text category={TextCategory.H3}>Chat</Text>
+                <Button
```
Comment (illia-dobro) [inline mobile/src/bundles/chat/screens/chat-list/chat-list.tsx:106]: We will have a message about approval on profile pages, where the logout button is. Logout button looks in the right place, but how to combine it with the message?

**Review by katerynakostikova — APPROVED** [review]

**Comment by katerynakostikova** [comment]
> Don't know if it's a good idea to create a common data API, and use it to employer and talent. We create unnecessary connexion between those bundles adn violated open/closed principles in talent store. As for me, we should use employer slice as a separate slice. Better for logic and scaling application. @katerynakostikova

We use the same endpoints for both roles, so I decided to same actions for both roles, otherwise, we'd have a lot of code repeating. Both suggestions have some props and cons, for now lets leave as it is, to make sure we're able to finish everything till final demo

File: mobile/src/bundles/chat/screens/chat-list/chat-list.tsx (hunk 2 above)
Comment (anogire) [reply mobile/src/bundles/chat/screens/chat-list/chat-list.tsx:106]: Then it's probably better to remove Logout from this page.
From here, you can go to the bottom of the navigation to Chat or Candidates and then log out.

**Review by illia-dobro — APPROVED** [review]

**Pull request merged**